test/openAPI/features/db_read.feature

```diff
@@ -1,21 +1,29 @@
+@method=GET @endpoint=/database/{id}
 Feature: API endpoint that allows users to retrieve Digital Registries database information with schema versions.
-  Request endpoint: GET /database/{id}/
 
-  Background:
+  @smoke 
+  Scenario: User successfully obtains Digital Registries database information with schema version smoke type test
     Given User wants to get the database information of Digital Registries with schema version
+    When User sends GET request with given Information-Mediator-Client header and "1" as id
+    Then User receives a response from the /database/id endpoint
```

Comment:
I think if we should name ```/database/id``` differently since two endpoints are the same (delete, get). What do you think @KarolinaKopacz